CHANGES.md

```diff
@@ -1,10 +1,413 @@
 Release Changes List
 ====================
 
+TODO:
+o add late-breaking entries
+o re-read rendered version with fresh eyes
+o check that hyperlinks work
+o re-spellcheck
+
+version 1.24.0
+==============
+
+Twenty-seventh public release of Chapel, March 18, 2021
+
+Highlights (see subsequent sections for further details)
+--------------------------------------------------------
+* language:
+  - significantly improved the syntax and semantics of overloading operators
+  - clarified the relationship between various type conversion mechanisms
+  - improved ergonomics for using nilable classes within conditionals and loops
+  - improved the definition and handling of tertiary methods
+  - unified the handling of `out` intents and `return` statements
+  - added initial support for defining constrained generic interfaces
+* performance:
+  - improved the performance of `--cache-remote` and enabled it by default
+  - added a new optimization that automatically aggregates communications
+  - improved the generality and reporting of the auto-local-access optimization
+  - dramatically reduced the overheads of tracking a domain's arrays
+  - improved performance for various scan operations on 1D arrays
+  - improved compile times for programs using zippered `forall` loops
+* libraries:
+  - refactored the 'Memory' module and added a new 'Initialization' sub-module
+  - added a new 'UnrolledLinkedList' module
+  - improved the flexibility of initializing lists
+  - added support for 'PrivateDist' arrays of non-nilable classes
+  - updated the 'LinearAlgebra' module to use 0-based indexing
+  - improved the generality of `sort()` in the 'Sort' module
+  - refactored the documentation and organization of the standard libraries
+* misc:
+  - added syntax highlighting for 'geany' and LaTeX
+  - significantly improved `ofi` portability, performance, and selection
+  - closed all remaining known memory leaks
+  - significantly improved the readiness of the LLVM-based compiler back-end
+  - removed Python 2 dependences in favor of Python 3
+  - improved several launcher choices and options
+
+Packaging / Configuration Changes
+---------------------------------
+* a C++11 compiler is now required to build the Chapel compiler
+* added `bundled` options for several `CHPL_*` environment variables
+* added a Lua/Lmod module file for Chapel on HPE Cray XC and EX systems
+* expanded support for HPE Cray EX systems  
+  (see https://chapel-lang.org/docs/1.24/platforms/cray.html#getting-started-with-chapel-on-hpe-cray-ex-systems)
```

Comment:
Sorry, I should have remembered that I'd sent you this link when I updated the docs yester-eve. The link for this is now https://chapel-lang.org/docs/1.24/platforms/cray.html#getting-started-with-chapel-on-cray-xc-or-hpe-cray-ex-systems. (Just the tag part changed, after the '#' character.)